.github/workflows/remove-stale-branches.yml

```diff
@@ -0,0 +1,26 @@
+name: remove-stale-branches
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+  # Allow workflow to be manually run from the GitHub UI
+  workflow_dispatch:
+
+jobs:
+  remove_stale_branches:
+    runs-on: ubuntu-latest
+    name: remove stale branches
+    steps:
+      - name: remove stale branches
+        uses: drew-dectech/github-action-inactive-branches@7aca3178001bdbff984c3eeb3eeb0f972bd75ae4 # v1.2.1
```

Comment:
Is this now our convention for specifying versions of GitHub actions? To use a digest instead of a version number? What is the advantage of this? And what happens when Renovate or Dependabot will update this version? That comment at the end will then not be correct anymore right? Or will it be removed even then?

Reply:
It's the recommended approach to get as much consistency and control in pipeline runs:
* The action now refers to the commit hash of the action repository
* Even specific version tagging might not have the same result as the version tag can be modified later.

This common action notation convention is known by both renovate and dependabot. When a newer version of the GitHub action is released, the version comment will be updated and the commit hash will be updated to refer to the HEAD of that version tag.

Reply:
And yes, I was planning on doing that on all pipelines :)

Reply:
Ah cool. Didn't know that. Nice. :-)